manifests/init.pp

```diff
@@ -144,6 +144,18 @@
 # $candlepin_conf_file::          Configuration file location for candlepin
 #                                 type:Stdlib::Absolutepath
 #
+# $manage_repo::                  Whether to manage the yum repository
+#                                 type:Boolean
+#
+# $repo::                         Which yum repository to install. For example latest or 3.3
```

Comment:
This will be a little strange for users of this module given its the Katello version to be specified and not the Candlepin version (and not necessarily obvious unless we rename the parameter).